packages/indexer-common/src/transactions.ts

```diff
@@ -420,6 +423,42 @@ export class TransactionManager {
     })
   }
 
+  async monitorIsHorizon(
```

Comment:
I think this is not necessary. It seems to me that the monitor functions on `TransactionManager` are not being used at all.

Reply:
I can't see why we have the other monitor functions here, perhaps it's dead code that can be removed? I would not add the horizon monitor here.

Reply:
seems to me the `TransactionsManager` handles write ops and the `NetworkMonitor` handles read ops, it's likely previously we had just one for both and the code got duplicated?